custom/icds_reports/management/commands/sql_scripts/build_migration_table_jan.sql

```diff
@@ -0,0 +1,22 @@
+INSERT INTO "icds_dashboard_migration_forms" (
```

Comment:
Only For jan.

Reply:
How do you plan to run this? May be nice to add it to the management command above so you arent executing in a raw sql shell. You can have it decide whether to run this or the normal aggregation based on the month